README.md

```diff
@@ -67,13 +67,13 @@ end
 
 
   kind = case year
-         when 1850..1889 then "Blues"
-         when 1890..1909 then "Ragtime"
-         when 1910..1929 then "New Orleans Jazz"
-         when 1930..1939 then "Swing"
-         when 1940..1950 then "Bebop"
-         else "Jazz"
-         end
+  when 1850..1889 then "Blues"
+  when 1890..1909 then "Ragtime"
+  when 1910..1929 then "New Orleans Jazz"
+  when 1930..1939 then "Swing"
+  when 1940..1950 then "Bebop"
+  else "Jazz"
+  end
```

Comment:
@cfabianski I use that as a workaround for our `if` indentation cop actually, it moans about

```ruby
result = if something
  :yes
else
  :no
end
```

Whereas it doesn't moan about:

```ruby
result = 
  if something
    :yes
  else
    :no
  end
```

I'm guessing those guides are suggesting that style to match the "indent when as deep as case", without having the original code in our example where `when` has a load of indentation.